iree/compiler/Codegen/LLVMGPU/KernelConfig.cpp

```diff
@@ -103,7 +103,7 @@ static LogicalResult setContractConfig(FuncOp entryPoint, linalg::LinalgOp op) {
       {tileX / workgroupSize[1], tileY / workgroupSize[0]});
   tileSizes.push_back(invocationLevelTs);  // Thread level.
   return setOpConfigAndEntryPointFnTranslation(
-      entryPoint, op, tileSizes, /*nativeVectorSize=*/ArrayRef<int64_t>{},
+      entryPoint, op, tileSizes, /*nativeVectorSizes=*/ArrayRef<int64_t>{},
```

Comment:
I think the reason this was wonky is because the name in the definition doesn't match the name in the declaration. Could you fix that as well please?

https://github.com/google/iree/blob/fd8a769f8ca88eaf358d472e6d65f0616c22a52f/iree/compiler/Codegen/Utils/Utils.cpp#L85